core-services/image-mirroring/openshift/mapping_origin_4_12

```diff
@@ -93,6 +93,7 @@ registry.ci.openshift.org/origin/4.12:configmap-reloader quay.io/openshift/origi
 registry.ci.openshift.org/origin/4.12:console quay.io/openshift/origin-console:4.12 quay.io/openshift/origin-console:4.12.0
 registry.ci.openshift.org/origin/4.12:console-operator quay.io/openshift/origin-console-operator:4.12 quay.io/openshift/origin-console-operator:4.12.0
 registry.ci.openshift.org/origin/4.12:console-plugin-demo quay.io/openshift/origin-console-plugin-demo:4.12 quay.io/openshift/origin-console-plugin-demo:4.12.0
+registry.ci.openshift.org/origin/4.12:console-plugin-template quay.io/openshift/origin-console-plugin-template:4.12 quay.io/openshift/origin-console-plugin-template:4.12.0
```

Comment:
not sure why is this in the `mapping_origin_4_12` file. I think it should be in the `mapping_origin_4_14`